examples/test_mass_matrix.py

```diff
@@ -1,21 +1,37 @@
+"""
+USAGE
+=====
+
+$ ipython
+
+In [1]: run poisson_2d_mapping.py -t target -n 10 20 -s -c
```

Comment:
@yguclu Not certain about the intended usage of this file.

Reply:
See next comment